src/main/java/uk/gov/digital/ho/hocs/casework/domain/exception/RestResponseEntityExceptionHandler.java

```diff
@@ -75,7 +79,10 @@ public ResponseEntity handle(HttpMessageNotReadableException e) {
 
     @ExceptionHandler(Exception.class)
     public ResponseEntity handle(Exception e) {
-        log.error("Exception: {}", e.getMessage(), value(EVENT, UNCAUGHT_EXCEPTION));
+        Writer stackTraceWriter = new StringWriter();
+        PrintWriter printWriter = new PrintWriter(stackTraceWriter);
```

Comment:
Could we not do something like `e.getStackTrace().toString()` instead? Not sure if we need to do this as I think the Exception object already has everything we need.

Reply:
Unfortunately this just prints [Ljava.lang.StackTraceElement;@... I've done a quick Google, but there doesn't seem to be a simpler way to stringify the stacktrace.